subscribed_topics:
  point_cloud_internal_topic: synchronized/camera/depth/points
  depth_candidate_holes_topic: synchronized/camera/depth/candidate_holes
  rgb_candidate_holes_topic: synchronized/camera/rgb/candidate_holes

published_topics:
  synchronizer_unlock_topic: unlock_rgb_depth_synchronizer
  hole_detector_output_topic: /vision/holes_direction
  enhanced_holes_topic: /vision/enhanced_holes
